### PR TITLE
docs: add interactive game blog

### PR DESCRIPTION
**Pull request by templexxx**
## Summary

new blog about interactive game

### CODE REVIEW

**Comment by templexxx** [comment]
> There are some markdown formats error.

I can't see Vercel details, is it working now? @jolestar

File: docs/website/pages/blog/little_seasoned_ugly-interactive_game.en-US.mdx [hunk 1]
```diff
@@ -416,7 +416,7 @@ concurrent execution in the process.
 ----
 
 **Moe**: The easiest way to do this is to force the emulator to execute
-single-threadedly.
+in single-threaded.
```
Comment (templexxx) [inline docs/website/pages/blog/little_seasoned_ugly-interactive_game.en-US.mdx:419]: single-threaded is an adjective, right?

Comment (geometryolife) [reply docs/website/pages/blog/little_seasoned_ugly-interactive_game.en-US.mdx]: yeah, `in a single-threaded manner` 可以表示 `以单线程的方式`

Comment (templexxx) [reply docs/website/pages/blog/little_seasoned_ugly-interactive_game.en-US.mdx]: So there is something wrong with your change here. I've changed it to on a single thread.

**Review by jolestar — APPROVED** [review]

**Pull request merged**